src/main/java/org/jenkinsci/test/acceptance/controller/LocalController.java

```diff
@@ -374,10 +375,15 @@ private void diagnoseFailedLoad(Exception cause) throws IOException {
             }
 
             try {
-                Process jstack = new ProcessBuilder("jstack", String.valueOf(pid)).start();
+                File tmpFile = File.createTempFile("jstack-output", ".tmp");
+                tmpFile.deleteOnExit();
+                Process jstack = new ProcessBuilder("jstack", String.valueOf(pid))
+                        .redirectErrorStream(true)
+                        .redirectOutput(ProcessBuilder.Redirect.to(tmpFile))
+                        .start();
                 if (jstack.waitFor() == 0) {
                     StringWriter writer = new StringWriter();
-                    IOUtils.copy(jstack.getInputStream(), writer);
+                    IOUtils.copy(new FileInputStream(tmpFile), writer);
```

Comment:
Seems more logical to use `IOUtuls.readFileToString(tmpFile, Charset.defaultCharset())`